flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/Types.scala

```diff
@@ -196,4 +200,145 @@ object Types {
   def MULTISET[E](elementType: TypeInformation[E]): TypeInformation[util.Map[E, lang.Integer]] = {
     new MultisetTypeInfo(elementType)
   }
+
+  def fromJson(json: String): TypeInformation[_] = {
```

Comment:
Should we add this method? It's better to only use the py4j's ability for creating the Java Type. then we can get the benefits as follows:
- Reduce the dependency of  `table-planner module`, i.e. remove the dependency: org.json4s
- We do not need to maintain a new method `fromJson` when adding a new type in Java.
- Do not need to add `serializers` in python.

Can we try these efforts? And please let me know what do you think?

Reply:
Makes sense. Have updated the PR and addressed these issues.